docs/source/index.rst

```diff
@@ -61,6 +61,50 @@ For JBIG2 compression:
 * `jbig2enc <https://github.com/agl/jbig2enc>`_ for JBIG2 compression (and PyMuPDF 1.19.0 or higher)
 
 
+Installation
+------------
+
```

Comment:
I'm not sure if it makes sense to duplicate the whole section. I personally wouldn't do that.

Reply:
I think it's not too bad, if anything, maybe the README ought to refer to the docs for specifics.